src/app/modules/interviewQuestion/interviewQuestion.component.ts

```diff
@@ -34,6 +36,14 @@ export class InterviewQuestionComponent implements OnInit, OnDestroy {
     interval: any;
     contactHR: any;
     loading = true;
+    isSubmitted = true;
+    @HostListener('window:beforeunload', ['$event'])
+    onChange($event) {
+      if(this.isSubmitted) 
+      {
+          $event.returnValue='Your data will be lost!';
```

Comment:
change this message

If you reload your page, all your answers will be lost. are you sure you want to do this?